units/Fire_Reaper.cfg

```diff
@@ -0,0 +1,61 @@
+#textdomain wesnoth-loti
+[unit_type]
+    id=Fire Reaper
+    name=_"Fire Reaper"
+    [base_unit]
+        id=Reaper
+    [/base_unit]
+    advances_to="Fire Reaper"
+    level=5
+    hide_help=true
+    do_not_list=yes
+    image="units/undead/reaper.png~RC(9d9da2,161515,141313>orange)~RC(b2b2b2,171616,151514>yellow)"
```

Comment:
He'll change his colour back when an animation stats. You can prevent it by adding these colour changes as an `[effect]` that applies to `image_mod`.

Reply:
Done, thanks.